index.js

```diff
@@ -66,25 +66,44 @@ module.exports = function (PIXI)
                         sprite.dimension.w / resolution,
                         sprite.dimension.h / resolution
                     );
-                    var crop = new PIXI.Rectangle(
-                      0,
-                      0,
-                      sprite.dimension.w / resolution,
-                      sprite.dimension.h / resolution
-                    );
+
+                    var crop;
+                    if (Number(PIXI.VERSION.charAt(0)) >= 4) {
+                        crop = new PIXI.Rectangle(
+                            0,
+                            0,
+                            sprite.dimension.w / resolution,
+                            sprite.dimension.h / resolution
+                        );
+                    } else {
+                        crop = frame.clone();
+                    }
+
                     var trim = null;
 
                     //  Check to see if the sprite is trimmed
                     if (sprite.trim) {
-                        trim = new PIXI.Rectangle(
-                            sprite.trim.x / resolution,
-                            sprite.trim.y / resolution,
-                            sprite.trim.w / resolution,
-                            sprite.trim.h / resolution
-                        );
-
-                        frame.width = sprite.trim.w / resolution;
-                        frame.height = sprite.trim.h / resolution;
+                        if (Number(PIXI.VERSION.charAt(0)) >= 4) {
+                            trim = new PIXI.Rectangle(
+                                sprite.trim.x / resolution,
+                                sprite.trim.y / resolution,
+                                sprite.trim.w / resolution,
+                                sprite.trim.h / resolution
+                            );
+
+                            frame.width = sprite.trim.w / resolution;
+                            frame.height = sprite.trim.h / resolution;
+                        } else {
+                            trim = new PIXI.Rectangle(
+                                sprite.trim.x / resolution,
+                                sprite.trim.y / resolution,
+                                frame.width,
+                                frame.height
+                            );
+
+                            crop.width = sprite.trim.w / resolution;
+                            crop.height = sprite.trim.h / resolution;
```

Comment:
yes, but that's for >4, not for v3 (e.g. your else branch)? If the goal is backward compatibility nothing should change for pixi 3 users.